### ISSUE TITLE
Update mongodb dependency to fix warnings with NodeJS 14

### ISSUE THREAD

**Issue by sigalor** [strongloop/loopback-connector-mongodb#572]
After having updated to NodeJS 14.2.0, I noted the following warnings when starting my Loopback application:

```
(node:11476) Warning: Accessing non-existent property 'count' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:11476) Warning: Accessing non-existent property 'findOne' of module exports inside circular dependency
(node:11476) Warning: Accessing non-existent property 'remove' of module exports inside circular dependency
(node:11476) Warning: Accessing non-existent property 'updateOne' of module exports inside circular dependency
```

This issue originates from the `mongodb` NPM dependency. It is exactly the same issue as the one described in [this MongoDB forum entry](https://developer.mongodb.com/community/forums/t/warning-accessing-non-existent-property-count-of-module-exports-inside-circular-dependency/3183) and [this MongoDB Jira ticket](https://jira.mongodb.org/browse/NODE-2536) and was fixed a few days ago with the release of version 3.5.7.

In order to fix this, it would be nice if the `mongodb` dependency of `loopback-connector-mongodb` could be updated from 3.2.4 to 3.5.7.

**Comment by fedebaier** [comment]
Ping

**Comment by codepushr** [comment]
Why has this been closed? This is still an issue, people over at npm/mongodb fixed this in 3.5.7 (https://jira.mongodb.org/browse/NODE-2536) and we are still sitting at 3.2.4 ...
